File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1101.cs
using System;

class URI
{
    static void Main() {
        int m, n, soma, i, temp;
        while(true){
            string ons = System.Console.ReadLine().Trim();
            string[] ns = ons.Split(' '); 
            n = int.Parse(ns[0]);
            m = int.Parse(ns[1]);
            if(m<=0 || n<=0){
                break;
            }
            if(m>n){
                temp=n;
                n=m;
                m=temp;
            }
            soma=0;
            for(i=m;i<=n;i++){
                soma+=i;
                Console.Write("{0} ", i);
            }
            Console.Write("Sum={0}\n", soma);
        }
    }
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1101.lua
while(true) do
    n = io.read("*number")
    m = io.read("*number")
    res = ""
    if(m<=0 or n<=0) then
        break
    end
    if(m>n) then
        temp=n
        n=m
        m=temp
    end
    soma=0
    for i=m,n,1 do
        soma=soma+i
        res=res..i.." "
    end
    print(res.."Sum="..soma)
end